modules/pom.xml

```diff
@@ -23,6 +23,8 @@
         <aliases.xml.basedir>src/main/aliases</aliases.xml.basedir>
         <hibernate.search.version>${project.version}</hibernate.search.version>
         <hibernate.search.module.slot>${parsedVersion.majorVersion}.${parsedVersion.minorVersion}</hibernate.search.module.slot>
+        <infinispan.integration.slot>for-hibernatesearch-${hibernate.search.module.slot}</infinispan.integration.slot>
```

Comment:
@gustavonalle WDYT of this naming pattern to express the "provided for" idea?

The module below renders as

    <module name="org.infinispan.hibernate-search.directory-provider" slot="for-hibernatesearch-5.3" services="import" optional="true" />